README.md

```diff
@@ -45,15 +45,63 @@ The `api_type` is used to switch the behavior of the tap between using Salesforc
 To run discovery mode, execute the tap with the config file.
 
 ```
-> tap-salesforce --config config.json --discover > properties.json
+> tap-salesforce --config config.json --discover > catalog.json
 ```
 
 ## Sync Data
 
 To sync data, select fields in the `properties.json` output and run the tap.
 
 ```
-> tap-salesforce --config config.json --properties properties.json [--state state.json]
+> tap-salesforce --config config.json --catalog catalog.json [--state state.json]
 ```
 
+## Catalog, Extraction and Replication Types
+
+The included catalog file, `catalog.json`, is constructed to extract the Contact API and perform INCREMENTAL sync. 
+Refer to the [Singer spec](https://github.com/singer-io/getting-started/blob/master/docs/SYNC_MODE.md#replication-method) for 
+more information about replication types.
+
+By default, discovery mode will produce a catalog with forces FULL_TABLE sync for all streams. 
```

Comment:
nitpick
~~By default, discovery mode will produce a catalog with forces FULL_TABLE sync for all streams.~~
By default, discovery mode will produce a catalog that forces a FULL_TABLE sync for all streams.